doc/api/events.md

```diff
@@ -1748,14 +1748,16 @@ added: v14.5.0
 
 This is not used in Node.js and is provided purely for completeness.
 
-#### `event.cancelBubble()`
+#### `event.cancelBubble`
 
 <!-- YAML
 added: v14.5.0
 -->
 
-Alias for `event.stopPropagation()`. This is not used in Node.js and is
-provided purely for completeness.
+* {boolean}
```

Comment:
Sorry, my previous suggestion was incomplete
```suggestion
* Type: {boolean}
```

Reply:
I also missed it. Thank you for your help.